src/pudl/transform/eia861.py

```diff
@@ -1216,6 +1216,51 @@ def core_eia861__yearly_sales(raw_eia861__sales: pd.DataFrame) -> pd.DataFrame:
     return _post_process(transformed_sales)
 
 
+@asset(io_manager_key="pudl_io_manager")
+def core_eia861__yearly_short_form(
+    raw_eia861__short_form: pd.DataFrame,
+) -> pd.DataFrame:
+    """Transform the EIA 861 Short Form table.
+    Transformations include:
+    * Drop primary key duplicates.
+    * Convert N/Y values to boolean
+    * Change NA BA codes to 'UNK'
```

Comment:
I think this bulletized list will not format correctly in RST unless you put a blank line between it and the line above it.  There also needs to be a blank line between the first line of the docstring, and subsequent lines.

Reply:
Thank you. Adding the blank lines worked. Quick question, after running the command to build the docs. There were about eight JSON files added to my changes. Should I commit these JSON files? Or only commit the files I edited myself